plugins/source/hubspot/docs/overview.md

```diff
@@ -31,7 +31,11 @@ The following example sets up the HubSpot plugin, and connects it to a postgresq
 
 This is the specs that can be used by the HubSpot source Plugin.
 
+- `app_token` (`string`, optional, default: `HUBSPOT_APP_TOKEN` environment variable)
```

Comment:
```suggestion
- `app_token` (`string`)  (optional) (default: `HUBSPOT_APP_TOKEN` environment variable value)
```
as in other plugins.

Additionally, it might be useful to update other params & reolace stuff like `int` to `integer`